src/main/java/com/ericsson/ei/subscription/SubscriptionRepeatDbHandler.java

```diff
@@ -58,178 +59,220 @@
 
     // RepeatFlagHandling structure in MongoDb:
     // {
-    //    "_id" : ObjectId("5ac62b4ea4f87e29e8cc5915"),
-    //    "subscriptionId" : "subsA",
-    //               RequirementId corresponds to a Requirement List of matched Aggregated Objects Ids.
-    //                <RequirementId>   <AggrObjIds>
-    //    "requirements" : {"0" : [
-    //                                "11112",
-    //                                "72324",
-    //                                "72364",
-    //                                "72233",
-    //                                "71233"
-    //                           ],
-    //                     "1" : [
-    //                                "11112",
-    //                                "72324",
-    //                                "72364",
-    //                                "72233",
-    //                                "71233"
-    //                           ]
-    //                    }
+    // "_id" : ObjectId("5ac62b4ea4f87e29e8cc5915"),
+    // "subscriptionId" : "subsA",
+    // RequirementId corresponds to a Requirement List of matched Aggregated
+    // Objects
+    // Ids.
+    // <RequirementId> <AggrObjIds>
+    // "requirements" : {"0" : [
+    // "11112",
+    // "72324",
+    // "72364",
+    // "72233",
+    // "71233"
+    // ],
+    // "1" : [
+    // "11112",
+    // "72324",
+    // "72364",
+    // "72233",
+    // "71233"
+    // ]
+    // }
     // }
 
     /**
      * Function that stores the matched aggregatedObjectId to the database.
+     *
      * @param subscriptionId
      * @param requirementId
      * @param aggrObjId
      */
-    public void addMatchedAggrObjToSubscriptionId(String subscriptionId, int requirementId, String aggrObjId) {
-        LOGGER.debug("Adding/Updating matched AggrObjId: " + aggrObjId + " to SubscriptionsId: " + subscriptionId
-                + " aggrId matched list");
-
-        if (checkIfAggrObjIdExistInSubscriptionAggrIdsMatchedList(subscriptionId, requirementId, aggrObjId)) {
-            LOGGER.debug("Subscription: " + subscriptionId + " and AggrObjId, " + aggrObjId
-                    + " has already been matched." + "No need to register the subscription match.");
-            return;
-        }
-
-        if (!updateExistingMatchedSubscriptionWithAggrObjId(subscriptionId, requirementId, aggrObjId)) {
-            LOGGER.debug(
-                    "New Subscription AggrId has not matched, inserting new SubscriptionId and AggrObjId to matched list.");
-
-            insertNewMatchedAggregationToDatabase(subscriptionId, requirementId, aggrObjId);
-        }
+    public void addMatchedAggrObjToSubscriptionId(String subscriptionId,
+	    int requirementId, String aggrObjId) {
+	LOGGER.debug(
```

Comment:
Please fix this formatting with eiffel v4 formatter and break the string lines into smaller pieces.

Reply:
Done.